Redirect Firebase issues to FirebaseExtended/flutterfire (#42043)
flutter/flutter: .github/ISSUE_TEMPLATE/FlutterFire.md

---
name: FlutterFire
about: FlutterFire has been moved to [FirebaseExtended/flutterfire](https://github.com/FirebaseExtended/flutterfire).

>        All issues related to FlutterFire should be filed and will be

Reviewer: ```suggestion
       All issues related to FlutterFire should be filed there.
```

>        addressed there.

Reviewer: ```suggestion
```
